ocaml/file_formats/cmo_format.mli

```diff
@@ -28,12 +26,14 @@ type reloc_info =
 (* Descriptor for compilation units *)
 
 type compilation_unit_descr =
-  { cu_name: Compilation_unit.Name.t;   (* Name of compilation unit *)
```

Comment:
This should be `cu_unit` (and some variables of type `compilation_unit_descr` should probably be renamed from `unit` to `unit_descr` to avoid saying `unit.unit`; that indignity was why I didn't make this change earlier)